swaybar/render.c

```diff
@@ -53,21 +77,24 @@ static uint32_t render_status_line_error(cairo_t *cairo,
 
 	double text_y = height / 2.0 - text_height / 2.0;
 	cairo_move_to(cairo, *x, (int)floor(text_y));
+	choose_text_aa_mode(ctx, 0xFF0000FF);
 	pango_printf(cairo, font, output->scale, false, "%s", error);
 	*x -= margin;
 	return output->height;
 }
 
-static uint32_t render_status_line_text(cairo_t *cairo,
-		struct swaybar_output *output, double *x) {
+static uint32_t render_status_line_text(struct render_call *ctx, double *x) {
+	cairo_t *cairo = ctx->cairo;
```

Comment:
Tiny nit: `cairo` is not used until later, so this declaration could be moved to ~line 94.